internal/operators/manager_test.go

```diff
@@ -543,13 +543,20 @@ var _ = Describe("Operators manager", func() {
 		})
 
 		It("should be not valid if not enough disk space available for mce and odf", func() {
-			clusterHost = getMockHostWithDisks(int64(20), int64(20))
+			clusterHost = getMockHostWithDisks(int64(50), int64(50))
```

Comment:
Why did this change?

Reply:
This test wants to test a scenario where there is mce + odf => there should be at least 25 (ODF requirement) + 50 (+MCE requirement) = 75 GB. The 20 GB would fail for ODF even without MCE, so I increased it to 50

Reply:
I see, however the test states `"should be not valid if not enough disk space available for mce and odf"`

Reply:
I see now, we want ODF to pass but ODF+MCE to fail.
Nice catch